src/Geometry.h

```diff
@@ -27,6 +27,19 @@
 class Cmfd;
 
 /**
+ * @enum ringifyType
+ * @brief The method to use when subdividing cells into rings
+ */
+enum ringifyType{
```

Comment:
No problem!  But just so you know... I copied the style from `universeType` here:  https://github.com/mit-crpg/OpenMOC/blob/develop/src/Universe.h#L41

I even double checked that it didn't have a space there... but I just wanted to make mine look the same ;-)